presto-iceberg/src/main/java/com/facebook/presto/iceberg/TypeConverter.java

```diff
@@ -310,6 +310,9 @@ private static TypeInfo toHiveTypeInfo(Type type)
         if (DOUBLE.equals(type)) {
             return HIVE_DOUBLE.getTypeInfo();
         }
+        if (TimeType.TIME.equals(type)) {
+            return HIVE_INT.getTypeInfo();
```

Comment:
Should it be `HIVE_LONG`?

Reply:
Why would it be long? This is how it was merged internally...

Reply:
Time is defined to be in microseconds in Iceberg, so long can make sense. Even if time can only represent between 00:00 and 23:59.99999. There are 86,400 seconds in a day. 86.4B microseconds which exceeds integer size. So it should probably be a long.